about.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/index.css">
+    <link rel="stylesheet" href="./css/about.css">
+    <title>About</title>
+</head>
+<body>
+    <div class="container">
+        <a class="home" href="./index.html"><h3 class="name">Okewusi Oluwapelumi</h3></a>
+        <p class="title">Front end developer and designer</p>
+        <div class="main-container">
+            <div class="about-me">
+                <h3>About Me</h3>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Modi autem commodi omnis deserunt, atque delectus ratione? Ad ipsum, quis esse corrupti ut accusamus odit quam natus. Obcaecati non cupiditate modi? Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt non sequi debitis corrupti nisi obcaecati accusamus quisquam saepe. Illo, eius animi accusamus vero quidem natus? Ratione sed laboriosam suscipit tempore.</p>
+            </div>
+            <div class="bottom">
+                <div class="picture"></div>
+                <div class="skills-container">
+                    <div class="skills-title"><h3>Skill set</h3></div>
+                    <div class="skills">
+                        <div class="left-skills">
+                            <p>Figma</p>
+                            <p>Javascript</p>
+                            <p>React</p>
+                            <p>Mongodb</p>
+                        </div>
+                        <div class="right-skills">
+                            <p>HTML</p>
+                            <p>CSS</p>
+                            <p>Firebase</p>
+                        </div>
+                    </div>
```

Comment:
These list items are to represent and show your level of competence each skill. I can't seem to find your average percentage. Eg 
Html - 80% ±±±±=
CSS - 70%  ±±±==

Reply:
I didn't intend for them to show the average percentage, I just wanted to list the skills.

Reply:
Ok noted!